client/landing/jetpack-cloud/index.js

```diff
@@ -16,11 +24,31 @@ import 'components/environment-badge/style.scss';
 import 'layout/style.scss';
 import 'assets/stylesheets/jetpack-cloud.scss';
 
+const debug = debugFactory( 'jetpack-cloud' );
```

Comment:
`client/lib/user/user.js` will use `debug` with `'calypso:user'`, so if do use the Calypso user then we will need to make that injectable.

Reply:
Sorry I don't follow this. This just means that you will be able to use debug just this call in the console. and see where it happens.

Reply:
That's fair. I was thinking we could make it read `jetpack:user` or something like that, but it may be better to leave it as `calypso:user` since it is technically Calypso's user object.